src/smallmap_gui.h

```diff
@@ -27,6 +27,15 @@ void ShowSmallMap();
 void BuildLandLegend();
 void BuildOwnerLegend();
 
+/** Enum for how to include the heightmap pixels/colours in small map related functions */
+enum IncludeHeightmap {
+	IH_NEVER,                  ///< Never include the heightmap
+	IH_IF_ENABLED,             ///< Only include the heightmap if its enabled in the gui by the player
+	IH_ALWAYS                  ///< Always include the heightmap
+};
```

Comment:
Not a sticking point but just a suggestion: I think we tend to prefer class enums now for new things to limit scope polution. Declaration could look like e.g.:
```suggestion
enum class IncludeHeightmap {
	Never,                  ///< Never include the heightmap
	IfEnabled,             ///< Only include the heightmap if its enabled in the gui by the player
	Always                  ///< Always include the heightmap
};
```

Call sites would then use it as `IncludeHeightmap::Never` and so on.

Reply:
Will change it to scoped enum. I assumed that because all OpenTTD code i've seen so far does not do so, that I shouldn't as generally I personally prefer scoped enums anyway. I will remember this for the future.